### PR TITLE
Adding recipe for contrib-colormaps

### PR DESCRIPTION
**Pull request by jsignell**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

@story645 please comment to indicate that you are ok/not ok with being listed as a maintainer.

### CODE REVIEW

**Comment by story645** [comment]
Thanks for listing me as a maintainer! Totally ok with it. 🙂

**Comment by jsignell** [comment]
@conda-forge/help-python not sure what I'm doing wrong here.

**Comment by npavlovikj** [comment]
Please add `pip` in the `host` section.

**Comment by jsignell** [comment]
ah thanks!

File: recipes/contrib_colormaps/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=2.7
```
Comment (beckermr) [inline recipes/contrib_colormaps/meta.yaml]: This constraint on the python version is not needed. Ditto for the one below.

**Comment by beckermr** [comment]
@npavlovikj are you good here?

File: recipes/contrib_colormaps/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    - python
+
+test:
+  imports:
+    - contrib_colormaps
+
+about:
+  home: http://github.com/pyviz/contrib_colormaps
+  license: CC-BY-4.0
+  license_file: LICENSE.txt
```
Comment (npavlovikj) [inline recipes/contrib_colormaps/meta.yaml]: Please add `license_family: CC`.

**Comment by npavlovikj** [comment]
@beckermr , I just added comment about adding `license_family`, but other than that, the recipe looks good to me.

**Pull request merged**